### ISSUE TITLE
ARM - Reset bool using ?{} syntax

### ISSUE THREAD

**Issue by kraemv** [jasmin-lang/jasmin#626]
This program
```
export
fn main() {
    reg u32 a;
    reg bool cf;

    ?{ "==" = cf } = #CMP(a, a);
}
```
gives me the error 'typing error: implicit label OF need to be defined' when compiled with -arch arm-m4. Changing a to an u64 and compiling for x86 works.

**Comment by vbgl** [comment]
What version of the compiler are you using? (e.g., what does `jasminc -version` outputs?)

**Comment by vbgl** [comment]
Alright, I could reproduce the issue with version 2023.06.1. This has now been fixed, probably in https://github.com/jasmin-lang/jasmin/pull/594.

**Comment by vbgl** [comment]
Note that the fix is included in the latest release (2023.06.2).
